internal/v1/server_test.go

```diff
@@ -119,6 +119,7 @@ func startServerWithCustomDB(t *testing.T, url, provURL string, dbase db.DB, dis
 	require.NoError(t, err)
 
 	echoServer := echo.New()
+	echoServer.HideBanner = true
```

Comment:
Spent just an hour testing the code and I am already sick of that banner taking all the vertical space :-)

Reply:
Might be better to do this in a separate commit. I suggest putting the github comment into the commit message. Git is eternal, github might not be.